.github/workflows/components-contrib.yml

```diff
@@ -49,13 +49,12 @@ jobs:
           go-version: ${{ env.GOVER }}
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
-      - name: Install golangci-lint ${{ env.GOLANGCI_LINT_VER }}
-        if: matrix.target_arch != 'arm'
-        run: |
-          curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "${{ env.GOROOT }}/bin" "${{ env.GOLANGCI_LINT_VER }}"
-      - name: Run make lint
-        if: matrix.target_arch != 'arm' && matrix.target_os != 'windows'
-        run: make lint
+      - name: Run golangci-lint
+        if: matrix.target_arch == 'amd64' && matrix.target_os == 'linux'
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: ${{ env.GOLANGCI_LINT_VER }}
+          only-new-issues: true
```

Comment:
Please review this line. I found that it could miss some linter error in some cases because it checks the linter only in changed files.

Reply:
review->*remove

Reply:
Done.